tests/behat/features/fields.feature

```diff
@@ -228,3 +226,65 @@ Feature: Fields for Landing Page content type
     And I see field "Background Colour"
     And I should see an "select#edit-field-landing-page-bg-colour" element
     And I should see an "select#edit-field-landing-page-bg-colour.required" element
+
+  @api @suggest @javascript
+  Scenario: Request a landing page with an automated listing component via API
+    Given vocabulary "topic" with name "Topic" exists
+    And topic terms:
+      | name       | parent |
+      | Test Topic | 0      |
+
+    Given landing_page content:
+      | title                     | path                     | moderation_state | uuid                                | field_topic | field_node_primary_site | field_landing_page_summary | field_landing_page_bg_colour |
+      | [TEST] Landing Page title | /test-landing-page-alias | published        | 99999999-aaaa-bbbb-ccc-000000000000 | Test Topic  | Test Site               | Test Summary               | White                       |
```

Comment:
hi @asarfara,

Moving this test to tide_landing_page means to introduce tide_site as a suggested package in composer.json.